client/post-editor/editor-drawer/index.jsx

```diff
@@ -261,6 +262,17 @@ const EditorDrawer = React.createClass( {
 		);
 	},
 
+	renderCopyPost: function() {
+		// TODO: replace with currentPostTypeSupports( 'copy' );
```

Comment:
My understanding is that supported features are very granular (e.g. excerpts or even tags) and
while right now all post types would support copying, explicitly mapping it would prevent unnecessary rewrites later on.
How would I make this work? Well, supported features are retrieved via the REST API and I actually don't know what it would take, or how disruptive might be, adding a new one.